packages/beacon-node/src/execution/engine/interface.ts

```diff
@@ -187,6 +187,20 @@ export interface IExecutionEngine {
 
   getPayloadBodiesByRange(fork: ForkName, start: number, count: number): Promise<(ExecutionPayloadBody | null)[]>;
 
-  getBlobs(fork: ForkPostFulu, versionedHashes: VersionedHashes): Promise<BlobAndProofV2[] | null>;
-  getBlobs(fork: ForkPreFulu, versionedHashes: VersionedHashes): Promise<(BlobAndProof | null)[]>;
+  /**
+   * buffers is an optional parameter that can be used to pass pre-allocated buffers for the blobs and proofs
+   */
```

Comment:
should this rather be added on the property?

Reply:
just that the comment makes more sense as tsdoc of the property and not the whole function imo